src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -118,6 +118,7 @@ def two_groups_hyphens_two_replacements_with_suffix(
         "databricks.sql",
         "databricks.sqlalchemy",
     ),
+    "delta-lake": ("delta",),
```

Comment:
```suggestion
    "delta-spark": ("delta",),
```
did you intend to put `delta-spark` here?
https://pypi.org/project/delta-spark/